### ISSUE TITLE
Documentation - Accessibility tab cannot be initialized

### ISSUE THREAD

**Issue by antleblanc** [ovh/ovh-ui-kit#657]
## Description

Accessing to the following URL https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-action-menu-webcomponent--compact leds to an infinite initialization on the "Accessibility" tab.

Issue brought by: #656 

### Steps to reproduce

1. Go to: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-action-menu-webcomponent--compact
2. Reach `Accessibility` tab
3. See error

**Expected behavior**:

Accessibility `tab` should work.

**Actual behavior**:

Tab cannot initialize.

![documentation-accessibility](https://user-images.githubusercontent.com/428384/90288131-8317b700-de79-11ea-8cf1-eb29bb530a96.png)

**Frequency**:

5 (always)

### Additional information

**Browsers:**
- Chromium (Version 84.0.4147.105)
- Firefox 79.0

**Operating System:** GNU/Linux
**Distribution:** Ubuntu 18.04 (64-bit)

**Comment by AxelPeter** [comment]
This should be a configuration issue, I'll check if something is missing.

The warning is not related to the accessibility tab. Normally, the change from `|` to `/` has already been done.
